### ISSUE TITLE
Unable to close multiple selection MatSelect panel when TalkBack is enabled on Android

### ISSUE THREAD

**Issue by gabrielarcos** [angular/material2#12375]
#### Bug:
Unable to close multiple selection MatSelect panel when TalkBack tool is enabled on Android 7.0

#### What is the expected behavior?
MatSelect panel should close when tapping outside the panel.

#### What is the current behavior?
MatSelect panel it's not getting close when tapping outside the panel.

#### What are the steps to reproduce?
On the Android device go to Settings -> Accessibility, tap on Talkback and turn it on. Go to Chrome for Android an open the demo site https://material.angular.io/components/select/overview scroll down to "Select with multiple selection" example, tap on the Toppings select, select 1 or more options and tap outside the select panel.

#### What is the use-case or motivation for changing an existing behavior?
This is an Accessibility issue, user should be able to close the panel when using TalkBack accessibility tool

**Comment by josephperrott** [comment]
After doing some investigation further into this, this is definitely a tough one since we don't have the ability to know that talkback is being used or to intercept the back action happening, which is how you exit the native select menus.

We will need to work with our a11y team to determine how we can create a high quality a11y pattern for this.

It is worth noting that a TalkBack user can exit the MatSelect panel by continuing to navigate forward (or backward) through the list beyond the end of the selection panel.  Once outside, upon selecting an item once more it will close the selection panel as expected.

**Comment by jelbourn** [comment]
Confirmed that this bug still exists.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
